libs/src/api/reactions.js

```diff
@@ -0,0 +1,40 @@
+const { Base } = require('./base')
+
+class Reactions extends Base {
+  constructor (...args) {
+    super(...args)
+    this.submitReaction = this.submitReaction.bind(this)
+  }
+
+  /**
+   * Submit a user's reaction
+   * (represented by a numberic ID denoting one of 'heart' | 'fire' | 'party' | 'explode')
```

Comment:
I think it would be better not to include the reaction types here, since we'll forget to update this comment when we add more reactions